cves/CVE-2017-9787.yml

```diff
@@ -45,7 +45,8 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: A specific type of proxy created by Spring AOP could be used to
```

Comment:
A little more detail here would be great. What is Spring AOP and how does it relate to struts?

Reply:
I left in so little detail as you really didn't need to understand what Spring AOP was to understand the bug. All that mattered was it had a specific type of proxy. With that in mind I removed the reference to Spring AOP as it's not needed.